content/en/docs/concepts/architecture/nodes.md

```diff
@@ -330,6 +330,26 @@ the kubelet can use topology hints when making resource assignment decisions.
 See [Control Topology Management Policies on a Node](/docs/tasks/administer-cluster/topology-manager/)
 for more information.
 
+## Graceful Node Shutdown {#graceful-node-shutdown}
+
+{{< feature-state state="alpha" for_k8s_version="v1.20" >}}
+
+If you have enabled the `GracefulNodeShutdown` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/), then the kubelet attempts to detect the node system shutdown and terminates pods running on the node.
+Kubelet ensures that pods follow the normal [pod termination process](/docs/concepts/workloads/pods/pod-lifecycle/#pod-termination) during the node shutdown.
+
+When the `GracefulNodeShutdown` feature gate is enabled, kubelet uses [systemd inhibitor locks](https://www.freedesktop.org/wiki/Software/systemd/inhibit/) to delay the node shutdown with a given duration. During a shutdown kubelet terminates pods in two phases:
```

Comment:
Does this rely on the node running systemd?

Reply:
Yes, since this is using the systemd inhibitor locks. Do you think it makes sense to call out that this depends explicitly on node running systemd or is it implied with the sentence regarding `systemd inhibitor locks`?

Reply:
I added issue #25171 to track adding more documentation about graceful node shutdown.

There are style conventions for how to explain prerequisites, so the place to explain in detail what's required is there.